sklearn/utils/extmath.py

```diff
@@ -247,16 +275,24 @@ def randomized_svd(M, n_components, n_oversamples=10, n_iter=0,
         to ensure proper conditioning. The total number of random vectors
         used to find the range of M is n_components + n_oversamples.
 
-    n_iter: int (default is 0)
+    n_iter: int (default is 2)
         Number of power iterations (can be used to deal with very noisy
         problems).
 
+    power_iteration_normalizer: 'auto' (default), 'QR', 'LU', 'none'
+        Whether the power iterations are normalized with step-by-step
+        QR factorization (the slowest but most accurate), 'none'
+        (the fastest but numerically instable), or 'LU' factorization
```

Comment:
"...numerically unstable when the `n_iter` is large (e.g. typically 5 or larger)."